patches/components-safe_browsing-content-browser-client_side_model_loader.cc.patch

```diff
@@ -1,13 +1,12 @@
 diff --git a/components/safe_browsing/content/browser/client_side_model_loader.cc b/components/safe_browsing/content/browser/client_side_model_loader.cc
-index 7fbcb79da40246d9a77605dec15e85f041a45be4..fbd36247ea4432d32c1453720a4e743e8ad75dca 100644
+index 7fbcb79da40246d9a77605dec15e85f041a45be4..dab2fb89e86250c955a0708b398cb9cc9ddc8202 100644
 --- a/components/safe_browsing/content/browser/client_side_model_loader.cc
 +++ b/components/safe_browsing/content/browser/client_side_model_loader.cc
-@@ -61,7 +61,7 @@ std::string ReadFileIntoString(base::FilePath path) {
- const size_t ModelLoader::kMaxModelSizeBytes = 150 * 1024;
- const int ModelLoader::kClientModelFetchIntervalMs = 3600 * 1000;
- const char ModelLoader::kClientModelUrlPrefix[] =
--    "https://ssl.gstatic.com/safebrowsing/csd/";
-+    "https://static.brave.com/safebrowsing/csd/";
- const char ModelLoader::kClientModelNamePattern[] =
-     "client_model_v5%s_variation_%d.pb";
- #if !BUILDFLAG(FULL_SAFE_BROWSING)
+@@ -164,6 +164,7 @@ void ModelLoader::StartFetch(bool only_from_cache) {
+   // Start fetching the model either from the cache or possibly from the
+   // network if the model isn't in the cache.
+ 
++  BRAVE_DISABLE_FETCH
```

Comment:
we can block a network request here without introducing the patch. Can be done like this:
```
#include "components/safe_browsing/content/browser/client_side_model_loader.h"

#include "net/traffic_annotation/network_traffic_annotation.h"

#define NetworkTrafficAnnotationTag \
  IsCertificateError(net::OK);      \
  return;                           \
  net::NetworkTrafficAnnotationTag

#include "../../../../../../components/safe_browsing/content/browser/client_side_model_loader.cc"

#undef NetworkTrafficAnnotationTag
```

Reply:
Thanks @goodov , that's a very neat trick. Patch has been removed.